docs/modules/release-notes/pages/2023/v-3-26-legacy-jan-07-2023.adoc

```diff
@@ -0,0 +1,29 @@
+= v3.26| Legacy January 7, 2023
+:navtitle: v3.26 Legacy January 7, 2023
+
+These release notes cover Legacy Kobiton 3.26. As a reminder, in our November release, we started using 4.0+ as the numbering for releases of the new Kobiton, while Legacy Kobiton will continue with numbering starting with 3.25 and above. If you are using the new Kobiton, see the v4.1 release notes.
+
+If you have any questions or would like to update your Kobiton system, please send us an email at support@kobiton.com or contact your Account Executive.
+
+== New iOS and Android support
+
+Legacy Kobiton 3.26 supports the following:
+
+* iOS 15.7.1
+* iOS 16.1.1
+* iOS 16.1.2: supported except changing timezone
+* iOS 16.2: supported except changing timezone
+We also tested and support the following device model as stated:
```

Comment:
Since there isn't a newline between lines `15` and `16` AsciiDoc is rendering this as one sentence instead of two separate lines:

![Screenshot 2023-01-11 at 10 15 30 AM](https://user-images.githubusercontent.com/95643215/211885718-44e05d7c-b442-42f5-b948-24e9a1908191.png)

That being said, I'm actually going to convert this section into a table because I think it'll read better. Don't worry, I'll do this and push a commit - you don't have to. Here's the syntax for tables if you're curious: https://docs.asciidoctor.org/asciidoc/latest/tables/build-a-basic-table/